final_prep/README.md

```diff
@@ -19,27 +19,33 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [👍🏾] I have refactored my `section1` exercises to the best of my ability
+- [👍🏾] I have refactored my `section2` exercises to the best of my ability
+- [👍🏾] I have refactored my `section3` exercises to the best of my ability
+- [👍🏾] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [👍🏾] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [👍🏾] Study/Project work time
+- [👍🏾] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+- ![alt text](https://user-images.githubusercontent.com/79548116/116721018-7654d800-a9a2-11eb-8c03-42530d6ce9ae.png "Logo Title Text 1")
```

Comment:
Nice work with this- I'm glad you have your calendar linked up to Mod1, and I think you have an appropriate amount of time set aside for work. I'd encourage some color coding- it is hard to tell at a glance when you have personal time scheduled, or maybe personal events that should be highlighted. Also, I'd expect your weekends to fill in. I suggest you go ahead and block some work time, also block plenty of personal time, and leave some flexible to use as needed.

Reply:
Thanks! Just updated my calendar and made it more organized.

Reply:
done